table/block_based_table_factory.h

```diff
@@ -64,6 +80,7 @@ class BlockBasedTableFactory : public TableFactory {
 
  private:
   BlockBasedTableOptions table_options_;
+  mutable TailPrefetchStats tail_prefetch_stats_;
```

Comment:
I think it might be useful to export these stats and see a distribution (for later; not immediately needed).